cfgov/v1/tests/models/test_base.py

```diff
@@ -258,3 +260,21 @@ def test_assemble_csv(self):
                      "tester@example.com",
                      "{}".format(self.test_feedback.submitted_on.date())]:
             self.assertIn(term, test_csv)
+
+class TestMetaImage(TestCase):
+    def setUp(self):
+        self.social_sharing_image = mommy.prepare(CFGOVImage)
+
+    def test_meta_image_no_images(self):
+        page = mommy.prepare(
+            CFGOVPage,
+            social_sharing_image=None
+        )
+        self.assertIsNone(page.meta_image)
+
+    def test_meta_image(self):
```

Comment:
Could you describe what this test does, either in the name, or in a docstring?  It's unclear what behavior it is verifying.